frontend/src/components/Input/style.scss

```
@@ -0,0 +1,11 @@
+@import "@/common/variables.scss";
```

```suggestion
@import "~@/common/variables.scss";
```
scss에서 alias는 `~`로 시작해야되더라구요!
실행은 되긴 하는데... 안되는 경우도 있나봅니다 [관련 질문 글](https://stackoverflow.com/questions/34717203/webpack-with-sass-loader-scss-file-import-does-not-recognize-resolve-alias)

그러고보니 alias 설정 할 때 scss들을 적용하지 않았네요..😥

---

흠 저는 아무 문제 없이 작동했는데 고쳐야하는 문제일까요??

---

저는 일단 IDE에서 오류로 잡긴 합니다🤔

---

그러면 해당 부분 고치도록 하겠습니다 !